Api/MekanismReactor/ReactorController.lua

```diff
@@ -0,0 +1,82 @@
+ReactorController = {}
+ReactorController.__index = ReactorController
+
+local str_MesQueryStatusType = "Mes_QueryStatus"
+local str_MesSetInjectionRate = "Mes_SetInjectionRate"
+
+local str_ReactorControllerProtocolName = "Api_ReactorController"
+
+function ReactorController:new(obj_messageCommunicator, str_reactorControlSide)
+    if(obj_messageCommunicator == nil) then
+        error("obj_messageCommunicator" .. "value cannot be nil",2)
+    end
+
+    if(str_reactorControlSide == nil) then
+        error("str_reactorControlSide" .. "value cannot be nil",2)
+    end
+    obj_messageCommunicator:SetProtocolName(str_ReactorControllerProtocolName)
```

Comment:
debatable whether or not this should be mutable. 
maybe create a new communicator in the costructor? Atm i have no need for dependency injection.